zach_kaz/forloopbasic.py

```diff
@@ -0,0 +1,36 @@
+for i in range (1,151):
+    print(i)
+
+for i in range (5,1000001,5):
+    print(i)
+
+for i in range (1,101):
+    if i % 10 ==0:
+        print('Dojo')
+    elif i % 5==0:
+        print('Coding')
+    print(i)
```

Comment:
the output should not print 10 if the number is divisible by 10. //Look up fizz buzz. It's a classic interview question